### PR TITLE
RawSql instances for 9, 10, 11 and 12-column results

### PR DESCRIPTION
**Pull request by dgaw**
Adds extra RawSql instances discussed in #835

### CODE REVIEW

**Comment by parsonsmatt** [comment]
Looks great, thanks!

Can you add `@since 2.10.2` to these functions and instances as well as a ChangeLog entry for this?

**Comment by dgaw** [comment]
@parsonsmatt Thanks for reviewing. I've added the since comments and the changelog entry.

File: persistent/ChangeLog.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ## 2.10.2
 
 * Added `runMigrationQuiet` and `runMigrationUnsafeQuiet` to `Database.Persist.Sql.Migration` as safer alternatives to `runMigrationSilent`. [#971](https://github.com/yesodweb/persistent/pull/971)
-
   This functions as workaround/fix for: [#966](https://github.com/yesodweb/persistent/issues/966), [#948](https://github.com/yesodweb/persistent/issues/948), [#640](https://github.com/yesodweb/persistent/issues/640), and [#474](https://github.com/yesodweb/persistent/issues/474)
+* Added RawSql instances for 9, 10, 11 and 12-column results
```
Comment (parsonsmatt) [inline persistent/ChangeLog.md]:
```suggestion
* Added RawSql instances for 9, 10, 11 and 12-column results. [#961](https://github.com/yesodweb/persistent/pull/961)
```

Comment (dgaw) [reply persistent/ChangeLog.md]: Thanks for the suggestion.

**Pull request merged**